ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/application-traffic-recognition.j2

```diff
@@ -0,0 +1,91 @@
+{#
+ Copyright (c) 2023 Arista Networks, Inc.
+ Use of this source code is governed by the Apache License 2.0
+ that can be found in the LICENSE file.
+#}
+{# eos - application traffic recognition #}
+{% if application_traffic_recognition is arista.avd.defined %}
+!
+application traffic recognition
+{# categories #}
+{%     for category in application_traffic_recognition.categories | arista.avd.natural_sort('name') %}
+{%         if category.name is arista.avd.defined %}
+   category {{ category.name }}
+{%             for app_details in category.applications_and_services %}
+{%                 if app_details.service is  arista.avd.defined %}
+      application {{ app_details.name }} service {{ app_details.service }}
+{%                 else %}
+      application {{ app_details.name }}
+{%                 endif %}
+{%             endfor %}
+   !
+{%         endif %}
+{%     endfor %}
+{%     if application_traffic_recognition.port_range_sets is arista.avd.defined %}
+{%         for port_range in application_traffic_recognition.port_range_sets %}
+{%             if port_range.name is arista.avd.defined %}
+   field-set l4-port {{ port_range.name }}
+{%                 for port in port_range.ports %}
+      {{ port }}
+{%                 endfor %}
+   !
```

Comment:
It looks like it is rendered `{{ port }}, {{ port }}`  is it the same for you?

Reply:
No it gets rendered in a new line. This is what I can verify as per the config generated in the device documentation file as well.
So seems good to me.